website_img_dimension/static/src/js/website_img_dimension.js

```diff
@@ -0,0 +1,40 @@
+odoo.define('website_img_dimension', function(require) {
+    "use strict";
+    var Class = require('web.Class');
+    var web_editor_base = require('web_editor.base');
+
+    var TooltipManager = Class.extend({
+        init: function() {
+            var self = this;
+            web_editor_base.ready().done(function() {
```

Comment:
shouldn't you wrap everything inside this?

Reply:
If I wrap the thing in it I wouldn't be able to expose it.